### PR TITLE
develop -> master v0.6.0-rc3

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**

### CODE REVIEW

File: evm_loader/cli/Cargo.toml [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 authors = ["Cyber-Core Maintainers <maintainers@cyber-core.dev>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
+chrono = "0.4"
+thiserror = "1.0"
+num-derive = "0.3"
+num-traits = "0.2"
+num_enum = "0.5.1"
```
Comment (mich-master) [inline evm_loader/cli/Cargo.toml:13]: I'm not sure if these crates are currently used by the code. We should check it.

Comment (vasiliy-zaznobin) [reply evm_loader/cli/Cargo.toml:13]: It's somehow prevent to create a release?

Comment (mich-master) [reply evm_loader/cli/Cargo.toml:13]: No, it doesn't prevent us from creating the release.
It's just a comment, not requirement.

Comment (vasiliy-zaznobin) [reply evm_loader/cli/Cargo.toml:13]: Will you check it in develop branch?

Comment (mich-master) [reply evm_loader/cli/Cargo.toml:13]: I've just checked. These crates are unnecessary. We should remove them next release.

Comment (vasiliy-zaznobin) [reply evm_loader/cli/Cargo.toml:13]: OK. Please create an issue for that.

Comment (mich-master) [reply evm_loader/cli/Cargo.toml:13]: Reported.
https://github.com/neonlabsorg/neon-evm/issues/512

**Pull request merged**